jobs/build/buildvm-maint/scripts/gen_snapshot.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+echo "INSTALLED PYTHON PACKAGES"
+echo "========================="
+pip freeze
+
+echo ""
+echo "INSTALLED YUM PACKAGES"
+echo "======================"
+yum list installed
+
+echo ""
+echo "INSTALLED RUBY GEMS"
+echo "==================="
+gem list --local
+
+echo ""
+echo "DOCKER INFO"
+echo "==========="
+docker info
```

Comment:
docker info will change constantly since it lists freespace. Incompatible with a diff model.